### PR TITLE
Chore: add bundle id query param

### PR DESCRIPTION
**Pull request by llbartekll**
# Description

Resolves # (issue)

## Due Dilligence

* [ ] Breaking change
* [ ] Requires a documentation update

### CODE REVIEW

**Review by Cali93 — APPROVED** [review]
LGTM!

File: Sources/WalletConnectRelay/Dispatching.swift [hunk 1]
```diff
@@ -92,16 +98,22 @@ final class Dispatcher: NSObject, Dispatching {
 
 
     func protectedSend(_ string: String) async throws {
-        var isResumed = false
-        return try await withCheckedThrowingContinuation { continuation in
+        return try await withUnsafeThrowingContinuation { continuation in
+            var isResumed = false
+            let syncQueue = DispatchQueue(label: "com.walletconnect.sdk.dispatcher.protectedSend")
```
Comment (ignaciosantise) [inline Sources/WalletConnectRelay/Dispatching.swift:103]: is this label ok?

File: Example/RelayIntegrationTests/RelayClientEndToEndTests.swift [hunk 2]
```diff
@@ -74,38 +61,50 @@ final class RelayClientEndToEndTests: XCTestCase {
...
-//                Task(priority: .high) {  try await self.relayA.subscribe(topic: "ecb78f2df880c43d3418ddbf871092b847801932e21765b250cc50b9e96a9131") }
-//                subscribeExpectation.fulfill()
-//            }
-//        }.store(in: &publishers)
-//
-//        wait(for: [subscribeExpectation], timeout: InputConfig.defaultTimeout)
-//    }
-
-    func testEndToEndPayload() {
+    // test_bundleId_present - configured in the cloud to include bundleId for whitelisted apps
```
Comment (ignaciosantise) [inline Example/RelayIntegrationTests/RelayClientEndToEndTests.swift:64]: what about the failure case? Use a bundleId that is not added to the Cloud allowed ids

File: Sources/WalletConnectRelay/Dispatching.swift (hunk 1 above)
Comment (llbartekll) [reply Sources/WalletConnectRelay/Dispatching.swift:103]: i just keep it consistent

File: Example/RelayIntegrationTests/RelayClientEndToEndTests.swift (hunk 2 above)
Comment (llbartekll) [reply Example/RelayIntegrationTests/RelayClientEndToEndTests.swift:64]: I think this should be covered by the relay

**Pull request merged**